### PR TITLE
Remove `Invoke`'s post_dump

### PR DESCRIPTION
**Pull request by war-in**
Closes #

## Introduced changes

- 
- 

##

- [ ] This PR contains breaking changes

### CODE REVIEW

File: starknet_py/net/schemas/common.py [hunk 1]
```diff
@@ -95,6 +95,8 @@ def _deserialize(
 
 class TransactionTypeField(fields.Field):
     def _serialize(self, value: Any, attr: str, obj: Any, **kwargs):
+        if value.name == "INVOKE":
+            return "INVOKE_FUNCTION"
         return value.name if value is not None else ""
```
Comment (drknzz) [inline starknet_py/net/schemas/common.py:100]: Do you know why a check for `None` is here?

File: starknet_py/net/schemas/common.py [hunk 2]
```diff
@@ -95,6 +95,8 @@ def _deserialize(
 
 class TransactionTypeField(fields.Field):
     def _serialize(self, value: Any, attr: str, obj: Any, **kwargs):
+        if value.name == "INVOKE":
```
Comment (drknzz) [inline starknet_py/net/schemas/common.py:98]:
```suggestion
        if value == TransactionType.INVOKE:
```

File: starknet_py/net/full_node_client.py [hunk 3]
```diff
@@ -455,9 +455,7 @@ def _create_broadcasted_deploy_account_properties(transaction: DeployAccount) ->
 
 def _create_broadcasted_txn_common_properties(transaction: AccountTransaction) -> dict:
     broadcasted_txn_common_properties = {
-        "type": "INVOKE"
-        if transaction.type == TransactionType.INVOKE
-        else transaction.type.name,
+        "type": transaction.type.name,
```
Comment (drknzz) [inline starknet_py/net/full_node_client.py:458]: 👍

File: starknet_py/net/schemas/common.py (hunk 1 above)
Comment (war-in) [reply starknet_py/net/schemas/common.py:100]: No, but let's leave it

**Review by drknzz — APPROVED** [review]

**Pull request merged**